articles/language/file-structure.md

```diff
@@ -385,3 +385,28 @@ function DotProduct(a : Double[], b : Double[]) : Double {
     return accum;
 }
 ```
+
+
+## Internal Declarations
+
+User-defined types, operations, and functions can also be declared as *internal*.
+This means that they can only be accessed from within the Q# project that they are declared in.
+When a project is used as a reference, all of its non-internal declarations are made available, but trying to use an internal declaration from another project is an error.
+This is useful for writing modular code that can be reused by other parts of your project, but which can still be changed later without breaking other projects that might depend on it.
+
```

Comment:
A little confused by the phrases here. "When a project is used as a reference, all of its non-internal declarations are made available, but **trying to use an internal declaration from another project is an error**." -> "trying to use an internal declaration from another project **will produce** an error?" or it is an error to try to use an internal declaration from another project?

"This is useful for writing modular code", "this" means the above phrase?

Reply:
Removed the "but" before "which".

Reply:
1. My meaning is, trying to use an internal declaration from another project is an error (mistake), which implies that trying to use an internal declaration from another project produces an error (diagnostic).
2. I think by "this" I meant "internal declarations," so I updated it to say that.
3. I updated it with a possible alternative wording we could use - I removed the "which can" but re-added the "but", so the "can" doesn't need to be repeated.